Tasks/RunDistributedTests/RunDistributedTests.ps1

```diff
@@ -77,27 +80,50 @@ if([int]::TryParse($testConfiguration, [ref]$testConfigurationId)){}
  
 if([string]::Equals($testSelection, "testPlan")) 
 {
-    if($checkTestAgentCompatScriptLocationMemberExists)
+    if($checkCustomSlicingEnabledMemberExists)
```

Comment:
If the user has selected the checkbox in the ui, and the agent is old, do we not want to let the user know that he needs to update the agent to get the feature? Instead of silently ignoring it